### PR TITLE
[MM-35061] - Welcome Email: Missing spacing between icon and paragraphs

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Fix styling in the welcome email like space between icons and paragraphs

Screenshot 2021-05-03 at 18 54 59

#### Ticket Link
[MM-35061](https://mattermost.atlassian.net/browse/MM-35061?atlOrigin=eyJpIjoiYWIzZjZhMTQ0Zjg1NGRhY2E4NmMyNjVjNTdiN2IyZWIiLCJwIjoiaiJ9)

#### Release Note
```release-note
NONE

```

### CODE REVIEW

File: templates/cloud_welcome_email.html [hunk 1]
```diff
@@ -67,14 +67,14 @@
...
+                                                    style="height: 15px; width: 15px; transform: translateY(2px);" alt="c_avatar">
                                             </span>
                                             <span
-                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 600; font-size: 16px; line-height: 24px; color: #3D3C40;">
+                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 600; font-size: 16px; line-height: 24px; color: #3D3C40; margin-left: 8px;">
                                                 {{.Props.InviteInfo}}
                                             </span>
                                             <p
-                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 400; font-size: 14px; line-height: 20px; color: #3D3C40; margin-left: 18px">
+                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 400; font-size: 14px; line-height: 20px; color: #3D3C40; margin-left: 26px">
```
Comment (pablovelezvidal) [inline templates/cloud_welcome_email.html:77]: best practice to add semicolon at the end of the inline-style

File: templates/cloud_welcome_email.html [hunk 2]
```diff
@@ -106,15 +106,15 @@
...
                                                 </span>
                                             </span>
                                             <span
-                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 600; font-size: 16px; line-height: 24px; color: #3D3C40;">
+                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 600; font-size: 16px; line-height: 24px; color: #3D3C40; margin-left: 8px;">
                                                 {{.Props.DownloadMMInfo}}
                                             </span>
                                             <p
-                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 400; font-size: 14px; line-height: 20px; color: #3D3C40;  margin-left: 18px">
+                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 400; font-size: 14px; line-height: 20px; color: #3D3C40;  margin-left: 26px">
```
Comment (pablovelezvidal) [inline templates/cloud_welcome_email.html:117]: same here

**Review by marianunez — APPROVED** [review]

File: templates/cloud_welcome_email.html (hunk 1 above)
Comment (AGMETEOR) [reply templates/cloud_welcome_email.html]: Thanks Pablo. Fixed the issue.

File: templates/cloud_welcome_email.html (hunk 2 above)
Comment (AGMETEOR) [reply templates/cloud_welcome_email.html]: Thanks Pablo. Fixed the issue.

**Review by pablovelezvidal — APPROVED** [review]

**Pull request merged**